model/pipeline/src/main/proto/schema.proto

```diff
@@ -39,6 +40,7 @@ message Field {
   FieldType type = 3;
   int32 id = 4;
   int32 encoding_position = 5;
+  repeated Option options = 6;
```

Comment:
Yes, I thought about this but eventually landed on the field. Proto also does it on the field. You are saying something about the **field**, not about the field type. It's also nicely balanced with the options on schema, there you are saying something about the **type** (instead of having options on a row).

_Last week in a pair programming session, we were using the options hacked into metadata (that's in the type) and the driver found that also weird._

Note: To make the circle complete we need to introduce **options on LogicalTypes** (want todo this in a later PR). This makes options available for serializing the data needed for some logical types. This makes it possible to put the proto options from Enum's as well (although, not for EnumValues...).

Reply:
I agree having it on a field makes more sense. Then it is analogous to column metadata in a SQL table definition.

Reply:
I'm going to resolve this... I think the consensus is todo not on type.